R/betareg-tidiers.R

```diff
@@ -70,7 +70,9 @@ tidy.betareg <- function(x, conf.int = FALSE, conf.level = .95, ...) {
 #' @seealso [augment()], [betareg::betareg()]
 #' @export
 augment.betareg <- function(x, data = model.frame(x), newdata = NULL,
-                            type.predict, type.residuals, ...) {
+                            type.predict = c("response", "link", "precision", "variance", "quantile"),
```

Comment:
In addition to adding these arguments, we should use `arg_match()` to validate them. That is, you'll want to add a line:

```r
type.predict <- arg_match(type.predict)
```

in the first few lines of `augment.betareg()`.

Reply:
This makes sense, will add this once we are aligned on resolution(s) for your other comments.

Reply:
This has been added.  Resolving conversation.